README.md

```diff
@@ -18,6 +21,8 @@ Your group will be both a "Customer" and a "Designer".  First you'll prepare for
 
 ***Q1.1 Briefly describe your Mission:***
 
+Our mission is an umbrella that lights up when there is rain on the forecast for that day and flies above your head and follows you around. This would really fit into our lives because we wouldn't have to look at the weather app anymore or carry an umbrella around. We envision putting the umbrella next to the door. Then, you can forget about it until it lights up, which means that it will be rainy that day. As you walk out of the door, the umbrella will hover over you and follow you to your destination, keeping you dry without you having to hold it. If it's not already raining, it will still follow you but stay closed, and once it starts raining it will open to protect you. We wouldn't worry about forgetting it on accident since it follows you around automatically, and we wouldn't have to worry about checking the weather.
```

Comment:
Nice! I'm thinking of having a similar umbrella device synced to your phone's location such that it moves along with you.